core/src/themes/test/colors/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="UTF-8" />
-    <title>Theme</title>
+    <title>Palette</title>
```

Comment:
It seems like it might make sense to rename this directory too? (`core/src/themes` -> palette)

Reply:
This would be a much more significant change and I don't believe is the right change. Currently there are global mixins and theme-specific styles in the `core/src/themes` directory (for iOS and MD themes).